app/views/page/homePage.scala.html

```diff
@@ -26,12 +26,13 @@
 *@
 
 @main("Dr. Elephant", "dashboard") {
-  <div class="jumbotron">
-    <h2>Hello there, I've been busy!</h2>
-    <p>I looked through <b>@numJobsAnalyzed</b> jobs today.<br>
-      About <b>@numJobsSevere</b> of them could use some tuning.<br>
-      About <b>@numJobsCritical</b> of them need some serious attention!
-    </p>
-  </div>
-  @results
+<div class="jumbotron">
+  <h2>Hello there, I've been busy!</h2>
+  <p>I've looked through <b>@("%,d".format(numJobsAnalyzedLastDay))</b> jobs that finished in the last 24 hours.<br>
```

Comment:
Hmmm ok but on second thought the whole reason I wanted to change the wording was to convey that what the number actually represents is how many processed jobs finished in the last 24 hours, not how many jobs Dr. Elephant has looked at (for example, it's possible that Dr. Elephant burned through a large queue of old jobs so it's actually looked at significantly more jobs than what this number represents). How about:

> Hello there, I've been busy!
> I've looked at 100 jobs that finished in the last 24 hours.